ush/forecast_postdet.sh

```diff
@@ -519,11 +520,19 @@ EOF
       logi=logf${FH3}
       pgbi=GFSPRS.GrbF${FH2}
       flxi=GFSFLX.GrbF${FH2}
-      atmo=$memdir/${CDUMP}.t${cyc}z.atmf${FH3}.$affix
-      sfco=$memdir/${CDUMP}.t${cyc}z.sfcf${FH3}.$affix
-      logo=$memdir/${CDUMP}.t${cyc}z.logf${FH3}.txt
-      pgbo=$memdir/${CDUMP}.t${cyc}z.master.grb2f${FH3}
-      flxo=$memdir/${CDUMP}.t${cyc}z.sfluxgrbf${FH3}.grib2
+      if [[ $RUN == "gefs" ]]; then
```

Comment:
This should be temporary until GFS is updated to new COM structure.

Reply:
Yes.

Reply:
Maybe we should just make a `$HISTORY_DIR` and a `$MASTER_DIR` and set appropriately instead of all these if-blocks.

Reply:
I agree, however, it will add more variables. If you agree, I can do this changes.